modules/stanford_news_extras/stanford_news_extras.install

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Implements hook_install().
+ */
+function stanford_news_extras_install() {
+
+  // Get the default banner image
+  $filename = "stanford_news_extras_default_image.png";
+  $file_path = drupal_get_path('module', 'stanford_news_extras') . "/images/" . $filename ;
+  $image = file_get_contents($file_path);
+
+  //Create a database entry.
+  $file = file_save_data($image, "public://" . $filename, FILE_EXISTS_REPLACE);
+
+  // And save the file id for default_image_ft to use.
+  variable_set('stanford_news_banner', $file->fid);
+
+  // Set the DS region block values
+  stanford_news_extras_set_ds_region_blocks();
+}
+
+/**
+ * Implements hook_enable().
+ */
+function stanford_news_extras_enable() {
+}
+
+/**
+ * Implements hook_disable().
+ */
+function stanford_news_extras_disable() {
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function stanford_news_extras_uninstall() {
+
+  // Remove the DS Blocks variables for news extras
+  $settings = variable_get('ds_extras_region_blocks');
+  unset($settings['banner']);
+  unset($settings['banner_overlay']);
+  variable_set('ds_extras_region_blocks', $settings);
+}
+
+
+/**
+ * Set the DS region block values.
+ */
+function stanford_news_extras_set_ds_region_blocks() {
```

Comment:
Because this function is called from an update hook you should check to see if the settings already exist so you don't overwrite them.